model/project_ref.go

```diff
@@ -43,8 +43,7 @@ type ProjectRef struct {
 	DeactivatePrevious bool     `bson:"deactivate_previous" json:"deactivate_previous" yaml:"deactivate_previous"`
 	Tags               []string `bson:"tags" json:"tags" yaml:"tags"`
 
-	// TracksPushEvents, if true indicates that Repotracker is triggered by
-	// Github PushEvents for this project, instead of the Repotracker runner
+	// TracksPushEvents, if true indicates that Repotracker is triggered by Github PushEvents for this project.
 	TracksPushEvents bool `bson:"tracks_push_events" json:"tracks_push_events" yaml:"tracks_push_events"`
```

Comment:
I don't totally follow the change. Wouldn't we want to remove this variable altogether?

Reply:
I think we want to keep it because it's possible webhooks weren't able to be configured, in which case we set tracks push events to false. We could probably instead set RepotrackerDisabled to true if webhooks can't be enabled though.

Reply:
Edit: I think RepotrackerDisabled is trickier, because users are allowed to intentionally set it to true for their project, and so if we found a webhook we wouldn't know if the repotracker had been disabled by us and so should be re-enabled, or by the user. (Or we could disallow users to disable the repotracker, I don't see why you would do this really)